### PR TITLE
Fix gravitational constant explanation in Particle Physics

### PR DESCRIPTION
**Pull request by SquarePear**
Fixing a small mistake in my workshop.
Thanks for noticing it @quackduck!

### CODE REVIEW

**Comment by quackduck** [comment]
No problem! You're welcome.

**Comment by quackduck** [comment]
I recommend you provide units, otherwise the gravitational constant can be any number of your choice.

**Review by sampoder — APPROVED** [review]

File: workshops/particle_physics/README.md [hunk 1]
```diff
@@ -221,7 +221,7 @@ const SCALE = 0.001
 let particles = []
 ```
 
-In physics, `G` refers to the [Gravitational Constant](https://simple.wikipedia.org/wiki/Gravitational_constant), which is 6.67 times [e](https://en.wikipedia.org/wiki/E_(mathematical_constant)) to the power of -11. This constant is used as the basis for all sorts of fun equations.
+In physics, `G` refers to the [Gravitational Constant](https://simple.wikipedia.org/wiki/Gravitational_constant), which is 6.67 x 10<sup>-11</sup> (6.67e-11). This constant is used as the basis for all sorts of fun equations.
```
Comment (quackduck) [inline workshops/particle_physics/README.md:224]:
```suggestion
In physics, `G` refers to the [Gravitational Constant](https://simple.wikipedia.org/wiki/Gravitational_constant), which is 6.67 × 10<sup>-11</sup> (6.67e-11) m3 kg<sup>-1</sup>s<sup>-2</sup>. This constant is used as the basis for all sorts of fun equations.
```

**Review by quackduck — CHANGES_REQUESTED** [review]

File: workshops/particle_physics/README.md [hunk 2]
```diff
@@ -221,7 +221,7 @@ const SCALE = 0.001
 let particles = []
 ```
 
-In physics, `G` refers to the [Gravitational Constant](https://simple.wikipedia.org/wiki/Gravitational_constant), which is 6.67 times [e](https://en.wikipedia.org/wiki/E_(mathematical_constant)) to the power of -11. This constant is used as the basis for all sorts of fun equations.
+In physics, `G` refers to the [Gravitational Constant](https://simple.wikipedia.org/wiki/Gravitational_constant), which is 6.67 x 10<sup>-11</sup> (6.67e-11) N⋅m<sup>2</sup>⋅kg<sup>-2</sup>. This constant is used as the basis for all sorts of fun equations.
```
Comment (quackduck) [inline workshops/particle_physics/README.md:224]:
```suggestion
In physics, `G` refers to the [Gravitational Constant](https://simple.wikipedia.org/wiki/Gravitational_constant), which is 6.67 x 10<sup>-11</sup> N⋅m<sup>2</sup>⋅kg<sup>-2</sup> (which we'll write as 6.67e-11 in JS). This constant is used as the basis for all sorts of fun equations.
```

**Review by quackduck — CHANGES_REQUESTED** [review]

File: workshops/particle_physics/README.md (hunk 2 above)
Comment (quackduck) [reply workshops/particle_physics/README.md:224]: @SquarePear this might make it clearer.

**Review by quackduck — APPROVED** [review]

**Comment by quackduck** [comment]
Approved!

**Comment by quackduck** [comment]
@sampoder, this looks ready to merge.

**Pull request merged**